env/Lib/site-packages/colorama-0.4.3.dist-info/LICENSE.txt

```diff
@@ -0,0 +1,27 @@
+Copyright (c) 2010 Jonathan Hartley
```

Comment:
Please, don't include any third party source code. Only the changes you made in this repo.

Reply:
> 
> 
> Please, don't include any third party source code. Only the changes you made in this repo.

I don't know dude, I just need to learn. The only file is utils.py 
Cheers